### ISSUE TITLE
Should gridcell's selected attribute follow focus?

### ISSUE THREAD

**Issue by benbeaudry** [w3c/aria#1155]
According to the [gridcell](https://www.w3.org/TR/wai-aria-1.1/#gridcell)'s specs,
> A gridcell may be [...] selectable.

For single-selection containers, the ARIA specification about [aria-selected](https://www.w3.org/TR/wai-aria-1.1/#aria-selected) says:
>The selection normally follows the focus [...].

With the current implementation in Chromium, the selection follows the focus for a gridcell by default. However, in Firefox and Edge (the legacy one), the implementation makes no such assumption about a link unifying the two states.

I believe that Chromium engineers interpreted the _a gridcell may be selectable_ as a _a gridcell has an implicit aria-selected value of true or false, i.e. is always selected_ when they initially implemented the feature. My opinion is that the selection should **not** follow the focus for a gridcell and, even more, a gridcell should not have an implicit aria-selected value of true or false.

After discussing it with @aleventhal on a CL removing this behavior from Chromium, we concluded:

1. Chromium should do like Firefox and the legacy Edge on this matter.
2. The specs should be updated to reflect that we cannot and should not assume an implicit value of the aria-selected state for a gridcell. Also, we could update the specs of aria-selected to mention that the selection should only follow the focus on elements/roles that have an implicit aria-selected value of true or false.

What are your thoughts?

**Comment by JAWS-test** [comment]
According to the ARIA specification aria-selected is neither required nor implicit in gridcell. Thus the error is with Chrome. Only if aria-selected is explicitly specified, the status selected should be set
